### PR TITLE
Fix outdated label references and links

### PR DESCRIPTION
**Pull request by Kissaki**
The labels `good-first-issue` and `up-for-grabs` no longer exist. Instead, `good first issue` and `help wanted` exist.

Live link: https://learn.microsoft.com/en-us/contribute/content/dotnet/dotnet-contribute

### CODE REVIEW

**Comment by Kissaki** [comment]
#sign-off

File: Contribute/content/dotnet/dotnet-contribute.md [hunk 1]
```diff
@@ -43,8 +43,8 @@ Following these guidelines will ensure a better experience for you and for us.
 
 Choose an existing issue and address it. You can look at our [open issues](https://github.com/dotnet/docs/issues) list and volunteer to work on the ones you're interested in:
 
-- Filter by the [good-first-issue](https://github.com/dotnet/docs/labels/good-first-issue) label for, well, good first issues.
-- Filter by the [up-for-grabs](https://github.com/dotnet/docs/labels/up-for-grabs) label for issues appropriate for community contribution. These issues usually require minimal context.
+- Filter by the [good first issue](https://github.com/dotnet/docs/labels/good first issue) label for, well, good first issues.
```
Comment (gewarren) [inline Contribute/content/dotnet/dotnet-contribute.md:46]:
```suggestion
- Filter by the [good first issue](https://github.com/dotnet/docs/labels/good%20first%20issue) label for, well, good first issues.
```

File: Contribute/content/dotnet/dotnet-contribute.md [hunk 2]
```diff
@@ -43,8 +43,8 @@ Following these guidelines will ensure a better experience for you and for us.
...
+- Filter by the [help wanted](https://github.com/dotnet/docs/labels/help wanted) label for issues appropriate for community contribution. These issues usually require minimal context.
```
Comment (gewarren) [inline Contribute/content/dotnet/dotnet-contribute.md:47]:
```suggestion
- Filter by the [help wanted](https://github.com/dotnet/docs/labels/help%20wanted) label for issues appropriate for community contribution. These issues usually require minimal context.
```

**Review by gewarren — APPROVED** [review]

**Comment by gewarren** [comment]
Thanks @Kissaki!

**Review by mikekinsman — APPROVED** [review]

**Pull request merged**